### ISSUE TITLE
Increase statistics

### ISSUE THREAD

**Issue by vonsturm** [mppmu/gerda-mage-sim#13]
- [ ] K42 sur array

**Comment by annkasch** [comment]
There are some simulations missing:

- [ ] Pb-214 on the n+ surface

- [ ] Tl-208 in the mini shroud

- [ ] Pb-214 in the mini shroud

**Comment by gipert** [comment]
I'll add the nplus-Pb214 simulations. What do the other items mean? More statistics is needed? How much? The simulations are already there

**Comment by gipert** [comment]
Added nplus-Pb214 in 498a31a.

**Comment by annkasch** [comment]
Sorry. I used the gerda-pdfs-v0.4-alpha.tar.gz file from github and there the pdfs were not inside.

**Comment by gipert** [comment]
You're right, I don't know why.

**Comment by vonsturm** [comment]
Probably because of the missmatch of simulated primaries?? Dunno... we need to check this in the next pdf version

**Comment by annkasch** [comment]
There are some simulations missing:

- [ ]  K-40 larveto-pmt_top

- [ ] Bi-212 larveto-pmt top

**Comment by ThWester** [comment]
I would also like to have coincidences for

- [ ] K-40 larveto-pmt_top/bottom
- [ ] K-40 larveto coppershroud

:)

**Comment by gipert** [comment]
Ciao, we already tried to simulate in the PMTs but we got really low statistics. Having a decent number of events would require a lot of CPU hours, maybe @vonsturm can comment better as she did those simulations

**Comment by ThWester** [comment]
Yeah, I just checked my test simulations I did a while ago, and for the PMT it seems really bad. Coppershroud is a bit better (~factor 20), and they should be close to the pmts spectrum, i guess.
Still requires something like 1e11 events a.k.a. 30kCPUh, from my estimations :/

**Comment by BirgitSchneider** [comment]
I will start with larveto/coppershroud/K40/coin and 1e10 events. Let's see how this will look.
However I don't have much hope for pmt_top & pmt_bottom.

**Comment by vonsturm** [comment]
Increased stats in 
- [x] larveto-copper_shroud-K40-edep ( 67bf10a ) -> 1E10

**Comment by BirgitSchneider** [comment]
I already made the macros and started the simulations right after my post yesterday. They will run about 23 h. It's only 2.3 kCPU/h, but if we are going to increase the stats to 1e11, than I would like not to simulate it twice so save some CPU/h.

**Comment by gipert** [comment]
* [ ] `larveto/fibers/K40`

needs more statistics, something else @annkasch?

**Comment by vonsturm** [comment]
- [ ] larveto/copper_shroud/*
